packages/mason/lib/src/bundler.dart

```diff
@@ -10,7 +10,7 @@ final _binaryFileTypes = RegExp(
   caseSensitive: false,
 );
 
-final _hookFiles = RegExp('(pre_gen.dart|post_gen.dart|pubspec.yaml)');
+final _hookFiles = RegExp('(.*.dart|pubspec.yaml)');
```

Comment:
Need to add more tests around this functionality to ensure hook execution can occur after bundling and unbundling.